### ISSUE TITLE
arm-linux-gnueabihf builds broken in recent nightlies 

### ISSUE THREAD

**Issue by jedisct1** [rust-lang/rust#36883]
Nightlies since Nov 30 cannot compile anything any more on `nightly-arm-unknown-linux-gnueabihf`:

```
$ cargo new ok --bin
$ cd ok
$ cargo build

   Compiling ok v0.1.0 (file:///tmp/ok)
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-Wl,--as-needed" "-Wl,-z,noexecstack" "-L" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib" "/tmp/ok/target/debug/ok.0.o" "-o" "/tmp/ok/target/debug/ok" "-Wl,--gc-sections" "-pie" "-nodefaultlibs" "-L" "/tmp/ok/target/debug/deps" "-L" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib" "-Wl,-Bstatic" "-Wl,-Bdynamic" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/libstd-958ae23af751ccd2.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/libpanic_unwind-a9a59c59600d7fc2.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/libunwind-6c9d51e4a80592ac.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/librand-2bb4926cb0753fd8.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/libcollections-d6fc9dbb93241229.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/librustc_unicode-ccb09dfcfc8f1ea6.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/liballoc-d486feb737ebb44c.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/liballoc_jemalloc-ffb42151b6c9616e.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/liblibc-233ca0904a4b2a29.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/libcore-aad7393ea968a91e.rlib" "/home/j/.multirust/toolchains/nightly-arm-unknown-linux-gnueabihf/lib/rustlib/arm-unknown-linux-gnueabihf/lib/libcompiler_builtins-89be4beb34851b55.rlib" "-l" "dl" "-l" "rt" "-l" "pthread" "-l" "gcc_s" "-l" "pthread" "-l" "c" "-l" "m" "-l" "rt" "-l" "util"
  = note: /tmp/ok/target/debug/ok.0.o: In function `ok::main':
/tmp/ok/src/main.rs:2: undefined reference to `std::io::stdio::_print::he5122a484fb7616f'
/tmp/ok/target/debug/ok.0.o: In function `main':
ok.cgu-0.rs:(.text.main+0x3c): undefined reference to `std::rt::lang_start::h30fa64cd44e8a41a'
collect2: error: ld returned 1 exit status
```

**Comment by TimNN** [comment]
This looks suspiciously similar to #36793 in which case it should hopefully be fixed by #36866, which should be available in todays or tomorrows nightly.

**Comment by alexcrichton** [comment]
Ah yeah I believe this is indeed #36866, hopefully we'll actually get a new nightly tonight!
